lutris/util/script_thread.py

```diff
@@ -0,0 +1,45 @@
+import os
+import subprocess
+from threading import Thread
+
+from lutris.util.log import logger
+from lutris.gui import dialogs
+
+
+class ScriptThread():
```

Comment:
This is called ScriptThread and yet it doesn't subclass Thread ???\!!!

Reply:
This class is totally undocumented, what is it supposed to achieve???

Reply:
ScriptThread, executes a given script in another thread `start()` or on the caller thread with `join()` with a wrapper fn over `subprocess.call` to handle the exceptions in the new thread.
If u're talking about lutris Thread class, i didnt need any of its fns so thats why i opted for a new one if u're talking about python one sure that make sense i guess i'll make it a subclass
2 - i'll add the doc according to ur explenation

Reply:
Thread sub classed + doc